tests/forcetorque/ForceTorqueTest.cc

```diff
@@ -0,0 +1,44 @@
+#include <gtest/gtest.h>
+#include <yarp/os/Bottle.h>
+#include <yarp/os/BufferedPort.h>
+#include <yarp/os/LogStream.h>
+#include <yarp/os/Network.h>
+#include <gz/sim/TestFixture.hh>
+
+TEST(ForceTorqueTest, PluginTest)
+{
+  yarp::os::NetworkBase::setLocalMode(true);
+  
+  // Maximum verbosity helps with debugging
+  gz::common::Console::SetVerbosity(4);
+
+  // Instantiate test fixture
+  gz::sim::TestFixture fixture("../../../tests/forcetorque/model.sdf");
+
+  int iterations = 1000;
+  fixture.Server()->Run(true, iterations, false);
+
+  yarp::os::Port p;         
+  p.open("/read");  
+  yarp::os::Network::connect("/forcetorque/measures:o","/read");
+  yarp::os::Bottle b;        
+  p.read(b);         
```

Comment:
macOS and Windows tests are failing. Perhaps we should add a check that the read is successful, checking the return value of this function (see https://www.yarp.it/latest/classyarp_1_1os_1_1Port.html#a7b4eb018faba3837d50fffee0ea722ef). If it fails, it is possible that it is not able to read in time anything, as nothing is published between the connect and the read. Perhaps we can try to migrate this code to `multipleanalogsensorsclient` and then look again at this code? Tryng to fix it now before migrating to `multipleanalogsensorsclient` may be a waste of time.